scripts/local/db-get.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ -f reference/backup.sql.gz ];
+  then mv reference/backup.sql.gz reference/backup-prev.sql.gz;
```

Comment:
I'm confused by this bit. Seems like you want to keep the old version as a backup so you do not lose it. What's the reason? If you run this command twice then you overwrite it anyhow.

Reply:
Yeah, it's really just a light weight "oh crap I wanted to keep using that other db, but I just ran the db:get script, and it's gone now" safeguard. But I didn't think it was necessary to keep anything more than just the previous one. I guess it's not necessary at all, you could always download it directly from pantheon manually. It's something I've used before, but I'm not tied to in any way.

Reply:
It would probably be most useful if there was a major database change, but you are already working on a PR before that update happened. This way you could have both database states locally for different branches and import the one you need.

It's definitely an edge case though, so if you're proposing getting rid of this, I wouldn't fight it.

Reply:
Let's keep it for now. It doesn't hurt anything and offers a small assurance. But I may change this when we revamp local script in the future.